molecule/scenario_resources/Dockerfile.j2

```diff
@@ -73,7 +73,9 @@ dnf --assumeyes install \
   python3 \
   python3-dnf \
   python3-firewall \
+  python3-setuptools \
   selinux-policy \
+  selinux-policy-targeted \
```

Comment:
Selinux policy wasn't installed

Reply:
@dralley Typically SELinux wraps around the entire container, rather than running within it. So that's why they removed these packages with F31 I guess.

Reply:
Yup.  This is the bit that complains.  I suppose the default policy is different, so while it can set ```permissive``` mode it can't switch to the ```targeted``` policy.

Reply:
@dralley The textbook answer is that switching to targeted requires a reboot, with the relabeling occurring during the reboot.